docs/manual/external-tools.tex

```diff
@@ -366,6 +363,35 @@
 \end{Verbatim}
 
 
+\subsectionAndLabel{Troubleshooting}{buck-troubleshooting}
+
+Use \<--verbose 8> to see diagnostic output including a command line that
+ought to be equivalent to Buck's use of an in-process Java compiler.
+For example, you might run
+
+\begin{Verbatim}
+./buck clean
+./buck fetch //...
+./buck build --verbose 8 //app:main
+\end{Verbatim}
+
+However, the command lines that buck prints with \<-v 8> is \emph{not} what
+Buck actually executes.  Buck's invocation of the Checker Framework may
+produce errors that cannot be reproduced from the command line.  This is
+because Buck uses
+\href{https://github.com/facebook/buck/blob/main/src/com/facebook/buck/jvm/java/Jsr199Javac.java}{in-process
+  compilation}, potentially multi-threaded, and because Buck may use use
+\href{https://github.com/facebook/buck/blob/main/src/com/facebook/buck/util/ClassLoaderCache.java}{multiple
+  classloaders} that are cached.
+
+You can force Buck to run an external javac, thus behaving exactly like the
+command line that its diagnostics print.  Use the configuration flag \<-c
+tools.javac=\emph{path-to-javac}>, for example:
+
+\begin{Verbatim}
+./buck-exe clean && ./buck-exe fetch //... && ./buck-exe build //app:main -c tools.javac=$(which javac)
```

Comment:
nit: Here the binary is `buck-exe` and above it's `buck`. Probably, worth using the same name.